docs/source/quickstart/advanced.rst

```diff
@@ -52,6 +52,61 @@ multiple regions the same way as with the ``run`` command::
 A region column will be added to reports generated that include multiple regions to
 indicate which region each row is from.
 
+.. _scheduling-policy-execution:
+
+Scheduling Policy Execution
+---------------------------
+
```

Comment:
i don't think theres any docs extant on custodian deployment setups per se, ie this isn't per se scheduling execution, its a policy date filter, ie. scheduling execution is the notion of running periodically via a cron or similiar facility, which might be worth explaining to avoid the confusion, or retitle as Filtering Policy Execution by Date.